packages/cli/src/lib/project/PolywrapProject.ts

```diff
@@ -128,6 +129,18 @@ export class PolywrapProject extends Project<PolywrapManifest> {
     return language as PolywrapManifestLanguage;
   }
 
+  public async getBuildLanguage(): Promise<PolywrapBuildLanguage> {
+    const manifestLanguage = await this.getManifestLanguage();
+
+    if (manifestLanguage === "interface") {
+      throw new Error(`Cannot build an interface project.`);
+    }
+
+    return manifestLanguage === "wasm/typescript"
+      ? "wasm/javascript"
+      : manifestLanguage;
```

Comment:
I would put this in a utility function inside of `project/manifests/polywrap/languages.ts`. This mapping can be used elsewhere I would imagine.

Reply:
I think this is probably a PolywrapProject specific; just like `getManifestLanguage`, which is right above this definition. I'd even say this util method is `getManifestLanguage` + some conditions in regards to build. It isn't used anywhere outside of this context.

No strong opinions either, will move it if you feel it's more correct

Reply:
Sounds good to me, single use so no need to abstract farther right now